Src/coffeescript/cql-execution/src/elm/instance.coffee

```diff
@@ -19,4 +19,4 @@ module.exports.Instance = class Instance extends Expression
     obj = {}
     for el in @element
       obj[el.name] = el.exec(ctx)
-    obj
+    if @classType is "{urn:hl7-org:elm-types:r1}Quantity" then new Quantity(obj) else obj
```

Comment:
We should think about what other system classes we need to do this with.  Two that come to mind are `Code` and `Concept`.  Can you consider putting in a TODO or a switch statement with placeholders just to remind us for later?